lib/api/users/data/paginated_ids.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'paginated_ids.g.dart';
+
+@JsonSerializable(
+  explicitToJson: true,
+  fieldRename: FieldRename.snake,
+  includeIfNull: false,
```

Comment:
This is not necessary and should be removed for consistency (it only affects the `toJson` output anyways).

If the json response is missing nullable fields, they are always parsed as `null`, even without this flag.